test/suites/integration/httproute_path_match_test.go

```diff
@@ -8,31 +8,30 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	"github.com/samber/lo"
-	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"log"
 	"os"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"time"
 )
 
 var _ = Describe("HTTPRoute path matches", func() {
 	It("HTTPRoute should support multiple path matches", func() {
-		gateway := testFramework.NewGateway()
-		testFramework.ExpectCreated(ctx, gateway)
-
+		gateway := testFramework.NewHttpGateway()
 		deployment1, service1 := testFramework.NewHttpApp(test.HTTPAppOptions{Name: "test-v1"})
 		deployment2, service2 := testFramework.NewHttpApp(test.HTTPAppOptions{Name: "test-v2"})
-		pathMatchHttpRoute := testFramework.NewPathMatchHttpRoute(gateway, []*v1.Service{service1, service2})
+		pathMatchHttpRoute := testFramework.NewPathMatchHttpRoute(gateway, []client.Object{service1, service2}, "http")
 
 		// Create Kubernetes API Objects
 		testFramework.ExpectCreated(ctx,
+			gateway,
 			pathMatchHttpRoute,
 			service1,
 			deployment1,
 			service2,
 			deployment2,
 		)
-		time.Sleep(2 * time.Minute) //Need some time to wait for VPCLattice resources to be created
+		time.Sleep(3 * time.Minute) //Need some time to wait for VPCLattice resources to be created
 
```

Comment:
If we hope to make every test cases self-contained and test traffic for each test cases, we have to wait for a  FULL propagation delay to make sure config can fully propogate to Vpc Lattice DP, that's a long time, not sure how we can improve it, don't test traffic? or run each e2e test in parallel?